frontend/src/data/auth.ts

```diff
@@ -83,3 +58,14 @@ auth.onAuthStateChanged(newUser => {
 export const isAuthenticated = (): boolean => !!user;
 
 export const getUser = (): firebase.User | null => user;
+
+export async function executeAsyncFunctionIfErrReturnMessage(
+  inputFunction: () => Promise<firebase.auth.UserCredential | void>
+): Promise<string | null> {
```

Comment:
Type alias
```suggestion
): Promise<errMsg | null> {
```

Reply:
Or Error instance.

Reply:
Thanks, fixed with commits:
- [change executeAsyncFunctionIfErrReturnMessage in auth.ts return type](https://github.com/d9nchik/kpi_student/pull/6/commits/0c3025ad53b7d5423dee5214bb0afa5b5f8c7ee4)